DFE-Digital/sts-plan-technology-for-your-school
# Feat: update C&S git submodule

## Overview

Updates git submodule to bring in the latest [layout changes](https://github.com/DFE-Digital/sts-content-support/pull/158) from Content and Support

## How to test

- Navigate to a C&S page (`/content/...`)
- Confirm the Plan Tech cookie consent banner is displayed
- Confirm the feedback banner at the bottom of the page is **not** displayed
- Accept cookies
- Confirm the feedback banner is now displayed at the bottom of the page

## Checklist

Delete any rows that do not apply to the PR.
